vm/devices/storage/disk_nvme/src/lib.rs

```diff
@@ -22,17 +22,19 @@ use stackfuture::StackFuture;
 use std::future::Future;
 use std::io;
 use std::pin::Pin;
+use std::sync::Arc;
```

Comment:
it doesn't look like this Arc wrapping is doing anything?

Reply:
ah, I see, the underlying type is now Arc'd...

Reply:
This is another part necessary for Restore path.

Reply:
For this one (putting aside Clone question from namespace) - I'd actually prefer to merge it earlier. I already had few merge conflicts in this area.